src/app/shell/provider/provider-activities/create-activity/create-description-form/create-description-form.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, ElementRef, EventEmitter, OnInit, Output, ViewChild } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { MatAutocomplete, MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
+import { Observable } from 'rxjs';
+import { COMMA, ENTER } from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { map, startWith} from 'rxjs/operators';
+
+@Component({
+  selector: 'app-create-description-form',
+  templateUrl: './create-description-form.component.html',
+  styleUrls: ['./create-description-form.component.scss']
+})
+export class CreateDescriptionFormComponent implements OnInit {
+  
+  DescriptionFormGroup: FormGroup;
+  @Output() descriptionFormGroup = new EventEmitter();
```

Comment:
check namings in all components

events should say that the trigger handlers eg: onDescriptionChange|descriptionChanged etc